src/words.jl

```diff
@@ -74,3 +80,18 @@ Base.@propagate_inbounds function Base.setindex!(w::Word, v::Integer, n::Integer
     @assert v > 0 "All entries of a Word must be positive integers"
     return @inbounds w.ptrs[n] = v
 end
+
+"""
+    lcp(u::Word, v::Word)
+Returns the length of longest common prefix of two words (and simultaneously
+the index at which the prefix ends).
+"""
+function lcp(u::Word, v::Word)
```

Comment:
I'd prefer `longestprefix` or something more wordy

Reply:
Yes and no. `lcp` seems to be well known: cf. [LCP array](https://en.wikipedia.org/wiki/LCP_array).

Reply:
My personal preference:
`lcp` > `longestcommonprefix` > `longestprefix`
so just tell me which one. 😁

Reply:
then `longestcommonprefix` let it be